build.gradle

```diff
@@ -112,6 +113,7 @@ dependencies {
     testCompile 'org.hamcrest:hamcrest-all:1.3'
     testCompile 'junit:junit:4.12'
     testCompile 'org.apache.hadoop:hadoop-minicluster:2.2.0'
+    testCompile "org.mockito:mockito-core:1.+"
```

Comment:
I think we should peg this to a specific version and just update it whenever we do dependency updates.